File: SISMUEBLUM C#/DetalleVenta.cs
using System;
public class DetalleVenta{
	private Producto _producto;
	private double _cantidad;
	private double _porcentajeDescuento;
	
	public DetalleVenta(Producto producto, double cantidad){
		this._producto = producto;
		this._cantidad = cantidad;
	}
	
	public DetalleVenta(Producto producto, double cantidad, double porcentajeDescuento){
		this._producto = producto;
		this._cantidad = cantidad;
		this._porcentajeDescuento = porcentajeDescuento;
	}
	
	public Producto producto{get{return this._producto;} set{this._producto = value;}}
	public double cantidad{get{return this._cantidad;} set{this._cantidad = value;}}
	public double porcentajeDescuento{get{return this._porcentajeDescuento;} set{this._porcentajeDescuento = value;}}
	
	~DetalleVenta(){}
	
}

File: SISMUEBLUM JAVA/DetalleVenta.java
public class DetalleVenta{
	private Producto producto;
	private double cantidad;
	private double porcentajeDescuento;
	
	public DetalleVenta(Producto producto, double cantidad){
		this.producto = producto;
		this.cantidad = cantidad;
	}
	
	public DetalleVenta(Producto producto, double cantidad, double porcentajeDescuento){
		this.producto = producto;
		this.cantidad = cantidad;
		this.porcentajeDescuento = porcentajeDescuento;
	}
	
	public Producto getProducto () {return this.producto;}
	public void setProducto (Producto valor) {this.producto = valor;}
	public double getCantidad () {return this.cantidad;}
	public void setCantidad (double valor) {this.cantidad = valor;}
	public double getPorcentajeDescuento () {return this.porcentajeDescuento;}
	public void setPorcentajeDescuento (double valor) {this.porcentajeDescuento = valor;}
}